.chglog/config.yml

```diff
@@ -0,0 +1,29 @@
+style: github
+template: CHANGELOG.tpl.md
+info:
+  title: CHANGELOG
+  repository_url: https://github.com/ldelossa/gcl
+options:
+  commits:
+    sort_by: Scope
+  commit_groups:
+    group_by: Scope
+  header:
+    pattern: '^(.*):\s*(.*)$'
+    pattern_maps:
+      - Scope
+      - Subject
+  issues:
+    prefix: 
+      - "#"
```

Comment:
Is this right if we're not using github issues?

Reply:
You can still link to a github issue in the commit. We are techically supported to link GH issues to jira.